AGENDUM-57 Completed helper functions

mbowman21 opened the pull request:
currently holds:
- connecting to database
- disconnecting from database
- create tag field (which will be used by AGENDUM-36)

mbowman21 committed:
database helper functions

currently holds:
- connecting to database
- disconnecting from database
- create tag field (which will be used by AGENDUM-36)

creallfluharty commented on db_helper.py:
```diff
@@ -0,0 +1,24 @@
+from mongoengine import connect
+from DBClasses.classes import Field, ValueType
```
I think these are at `db_classes` right now, right?

mbowman21 replied:
yup just realized that as i was going through my own code

mbowman21 replied:
just fixed it!

creallfluharty commented on db_helper.py:
```diff
@@ -0,0 +1,24 @@
+def connectToDatabase():
+    host = "mongodb://localhost:27017/mongodb"
+    connection = connect(host=host)
+    if connection == None:
```
completely a non-blocker but just in case you were interested, in Python all `None`s are exactly the same object in memory so it is convention to check for `None`s with `if x is None`

mbowman21 replied:
fixed

creallfluharty commented on db_helper.py:
```diff
@@ -0,0 +1,24 @@
+    host = "mongodb://localhost:27017/mongodb"
+    connection = connect(host=host)
+    if connection == None:
+        raise Exception("Failed to connect to the database")
```
also not really an issue but generally `Exception` is considered a little too broad to be instantiated directly, though I'm not sure what the best alternative would be (`IOError` maybe?)

mbowman21 replied:
changed to ConnectionError

creallfluharty reviewed (changes requested):
only hold-up for me is the import

mbowman21 committed:
update import line

mbowman21 committed:
fixed == to is

mbowman21 committed:
changed the Exception to a ConnectionError

mbowman21 committed:
commented out createTagField

since it creates a circular code review with AGENDUM-56 and is a function used in AGENDUM-36 decided to comment it out for now.

creallfluharty committed:
AGENDUM-57 add connection manager

mbowman21 committed:
Delete db_helper.py
